webcompat/templates/home-page/form.html

```diff
@@ -15,9 +15,25 @@ <h3 class="wc-Title--s">Report Site Issue</h3>
       <div class="r-Grid r-Grid--withGutter">
         <div class="r-Grid-cell r-minM--1of2">
           <div class="wc-Form-group js-Form-group">
+            <div class="wc-Form-information">
+              {{ form.url.label(class_='wc-Form-label') }}
+              {% if form.url.errors %}
+                {% for error in form.url.errors %}
+                  <span class="wc-Form-helpMessage">{{ error }}</span>
+                {% endfor %}
+              {% endif %}
+            </div>
+            <span class="wc-Form-validation">
+            {{ form.url(class_='wc-Form-item required',
+               placeholder='e.g. https://example.com',
+               required=True, type='url', pattern='') }}
+            </span>
+          </div>
+
+          <div class="wc-Form-group js-Form-group">
             <fieldset class="{% if form.problem_category.errors %}is-error{% endif %}">
               <div class="wc-Form-information">
-                <label class="wc-Form-label">What seems to be the trouble? <span class="wc-Form-required">*</span></label>
+                <label class="wc-Form-label" for="problem_type">Type of bug <span class="wc-Form-required">*</span></label>
```

Comment:
@karlcow now I remember, I broke the pattern (and forgot). while running the related functional tests, changing the label to use only the python defined label breaks this test: reporting-non-auth.js > Problem type validation

https://github.com/webcompat/webcompat.com/blob/master/tests/functional/reporting-non-auth.js#L161

(side note, I don't understand why the validation is designed to work this way but that is a different topic; I am trying to preserve existing logic)

the dependent JS logic is in bugform.js, https://github.com/webcompat/webcompat.com/pull/1578/files#diff-ce241059c28a13e03bcd0829da8d4eb7R355

instead of using the old css selector (which broke because of the field reordering), it seemed simpler to use a different selector using the `for` attribute as a marker for placing the field validation error message.

so in the latest pr, the problem type radio buttons use the hard-coded html label. but the latter (tested another browser) radio buttons use the python defined label. there is a merge conflict note I have to resolve before another review.